packages/nimble-components/src/rich-text-editor/template.ts

```diff
@@ -1,6 +1,53 @@
-import { html } from '@microsoft/fast-element';
+import { html, ref, repeat, children, elements } from '@microsoft/fast-element';
 import type { RichTextEditor } from '.';
+import { toolbarTag } from '../toolbar';
+import { toggleButtonTag } from '../toggle-button';
+import type { EditorButton } from './models/editor-button';
 
 export const template = html<RichTextEditor>`
```

Comment:
The whitespace in this template is very difficult to read. If prettier is not helping you can disable it here. If you look at other template.ts files you can see they frequently disable prettier on the `export cont template = html` node so they can have sane formatting. Please do so to make the template clearly readable.

Reply:
Yes I have added prettier ignore comment in the template.